packaging_automation/tests/test_update_docker.py

```diff
@@ -5,14 +5,17 @@
 
 from ..common_tool_methods import (run, get_version_details, DEFAULT_ENCODING_FOR_FILE_HANDLING,
                                    DEFAULT_UNICODE_ERROR_HANDLER)
+from dotenv import dotenv_values
 from ..update_docker import (update_docker_file_for_latest_postgres, update_regular_docker_compose_file,
-                             update_docker_file_alpine, update_docker_file_for_postgres12, update_changelog,
-                             update_pkgvars, read_postgres_version)
+                             update_docker_file_alpine, update_docker_file_for_postgres12,
+                             update_docker_file_for_postgres13, update_changelog)
 
 BASE_PATH = os.getenv("BASE_PATH", default=pathlib2.Path(__file__).parents[2])
 TEST_BASE_PATH = f"{BASE_PATH}/docker"
 PROJECT_VERSION = "10.0.3"
-POSTGRES_VERSION = "14.0"
+POSTGRES_14_VERSION = "14.1"
+POSTGRES_13_VERSION = "13.5"
+POSTGRES_12_VERSION = "12.9"
```

Comment:
Do you plan to update all these tests every time a new PG version is released?

I do not think that it is wise to create a test that needs to updated 4 times a year. Can you make it so that the tests also read the versions from `pkgvars`?

Reply:
Tests are not for testing that specific version. They are just for testing the code so there is no need to update everytime postgres version changes